### PR TITLE
coreos-base/oem-packet: drop cloud-config

### PR DESCRIPTION
**Pull request by crawford**
Ignition (starting with 0.14.0) will apply this configuration
instead. This also depends on coreos-metadata supporting
PACKET_API_BASE, which will be present starting in 0.8.0.

### CODE REVIEW

File: coreos-base/oem-packet/oem-packet-0.0.5.ebuild [hunk 1]
```diff
@@ -16,14 +16,11 @@ S="${WORKDIR}"
 
 src_prepare() {
 	sed -e "s\\@@OEM_VERSION_ID@@\\${PVR}\\g" \
-	    "${FILESDIR}/cloud-config.yml" > "${T}/cloud-config.yml" || die
+		"${FILESDIR}/oem-release" > "${T}/oem-release" || die
 }
 
 src_install() {
-	into "/usr/share/oem"
-	dobin "${FILESDIR}/phone-home.sh"
```
Comment (dm0-) [inline coreos-base/oem-packet/oem-packet-0.0.5.ebuild]: Doesn't the service file that replaces this need to be installed?

Comment (crawford) [inline coreos-base/oem-packet/oem-packet-0.0.5.ebuild]: Fixed.

Comment (dm0-) [inline coreos-base/oem-packet/oem-packet-0.0.5.ebuild]: Did you push the branch?  I don't see a difference here.  The service still doesn't appear to get installed anywhere.

Comment (crawford) [inline coreos-base/oem-packet/oem-packet-0.0.5.ebuild]: That's weird. I did force push it...

Comment (dm0-) [inline coreos-base/oem-packet/oem-packet-0.0.5.ebuild]: I think you created a PR from a branch in this repo, but you pushed the update to your own forked repo.  That version looks okay, but does that now need an Ignition update?

Comment (crawford) [inline coreos-base/oem-packet/oem-packet-0.0.5.ebuild]: https://github.com/coreos/ignition/pull/346

**Pull request merged**